subsys/net/openthread/Kconfig.defconfig

```diff
@@ -31,6 +31,48 @@ choice OPENTHREAD_IMPLEMENTATION
 	default OPENTHREAD_LIBRARY if OPENTHREAD_LIBRARY_AVAILABLE
 endchoice
 
+if SOC_SERIES_NRF54LX
+
+#To be removed
```

Comment:
I suggest we convert these to TODO for better trackability. Bonus point for Jira-reference

# TODO: To be removed when <insert condition>

Reply:
out-of-scope.

This is not related to this PR.